src/components/DevTeamCards.js

```diff
@@ -2,30 +2,29 @@ import React from "react";
 import { Card, CardHeader } from "semantic-ui-react";
 
 const team = {
-  yair: "https://shpeuf.s3.amazonaws.com/public/team/yair.jpg",
   omar: "https://shpeuf.s3.amazonaws.com/public/team/omar.jpg",
-  victoria: "https://shpeuf.s3.amazonaws.com/public/team/victoria.jpg",
+  jesus: "https://shpeuf.s3.amazonaws.com/public/team/jesus.jpg",
+  isabella: "https://shpeuf.s3.amazonaws.com/public/team/isabella.jpg",
   ju: "https://shpeuf.s3.amazonaws.com/public/team/julia.jpg",
+  victoria: "https://shpeuf.s3.amazonaws.com/public/team/victoria.jpg",
   mateo: "https://shpeuf.s3.amazonaws.com/public/team/mateo.jpg",
-  lucas: "https://shpeuf.s3.amazonaws.com/public/team/lucas.jpeg",
-  isabella: "https://shpeuf.s3.amazonaws.com/public/team/isabella.jpg",
-  joel: "https://shpeuf.s3.amazonaws.com/public/team/joel.jpeg",
-  jesus: "https://shpeuf.s3.amazonaws.com/public/team/jesus.jpg",
-  temp: "../assets/images/placeholder.png"
+  elisa: "https://shpeuf.s3.amazonaws.com/public/team/elisa.png",
+  alex: "https://shpeuf.s3.amazonaws.com/public/team/alex.png",
+  alejandro: "https://shpeuf.s3.amazonaws.com/public/team/alex.png"
```

Comment:
Two identical links for alex & alejandro. All good otherwise

Reply:
fixed

<img width="457" alt="image" src="https://github.com/shpe-uf/SHPE-UF-CLIENT/assets/118920287/aa08cdde-e8ed-4fb6-8b90-3e2a3f1e80df">